modules/dicom_archive/php/dicom_archive.class.inc

```diff
@@ -174,7 +174,7 @@ class Dicom_Archive extends \NDB_Menu_Filter
     {
         $this->addBasicText('patientID', 'Patient ID');
         $this->addBasicText('patientName', 'Patient Name');
-        $this->addSelect('site', 'Site', \Utility::getSiteList());
+        $this->addSelect('site', 'Site', \Utility::getSiteList(), array('id'=>'Site'));
```

Comment:
i think it might be better to change the test script rather than add this kind of arbitrary seeming argument (even though i think i was the one who suggested this lol sorry)
@kongtiaowang 
would it be ok to change [this line](https://github.com/aces/Loris/blob/minor/modules/dicom_archive/test/dicom_archiveTest.php#L32) to `static $site = "select[name='site']";`? can it be done in this PR or does it have to be done in a separate PR for this to pass travis?

Reply:
yeah i had the same thought lol but thought this was less of a change?? not sure

Reply:
@davidblader how did you get `static $site = "select[name='site']"` as the element identifier?

Reply:
`document.querySelector()` uses CSS selectors to retrieve DOM elements, which come in a lot of different forms but for this in particular i used `element[attr='value']` (e.g. find the first `select` element with a `name` attribute equal to `site`) which seems to be unique enough for this case

https://www.w3schools.com/cssref/css_selectors.asp

Reply:
thank u. @kongtiaowang is also good with this change